### ISSUE TITLE
[bug] No navigation on mobile

### ISSUE THREAD

**Issue by Mara-Li** [TuanManhCao/digital-garden#2]
On mobile, the navigation bar disappear, so I can't switch file/folder. 
![image](https://user-images.githubusercontent.com/30244939/169657895-d40340c0-cf61-44a1-a12d-820ec8a1cbf5.png)
Also, there is no graph view either

**Comment by ghost** [comment]
I've been searching everywhere for a solution to this bug and it seems it has to do with SSR in Next.js. Here are a few links:

https://stackoverflow.com/questions/71660743/server-side-rendering-broken-after-upgrading-to-mui-5-with-next-js

https://mui.com/base/react-no-ssr/

**Comment by ghost** [comment]
@TuanManhCao Do you need help with this? I'm willing to collaborate with you the best I can.

**Comment by TuanManhCao** [comment]
Right, I'm a bit tight in terms of time right now. But as this repo is getting more popular now, I'll take some time to look at this soon.
